### DISCUSSION TITLE
Rofi theme acting weird

### DISCUSSION THREAD

**Discussion by mstanciu552** [davatorium/rofi#1434] [Q&A] [answered]
I've been using a rofi theme for a while and it was great.
But today I upgraded many packages and the theme broke somehow.
![rofi_theme](https://user-images.githubusercontent.com/34579048/132137968-1007a8dc-c660-417e-ac2d-da4566814776.png)

The background around the selected element is not green...
Can anyone help?
Here is the config: https://github.com/mstanciu552/dotfiles/tree/main/rofi

**Comment by vredesbyyrd** [comment, marked as answer]
Please see the pinned issue #1397.

**Reply by mstanciu552** [reply]
Thank you so so much...I hated how it looked and I use rofi for everything.

**Reply by DaveDavenport** [reply]
great it solved it.
